docs/source/guides/install/unraid.rst

```diff
@@ -1,3 +1,14 @@
 Unraid
 --------------
-You can install our :unraid:`unraid community apps <community/apps?q=ytdl-sub#r>` through the  `Unraid Community Apps plugin <https://unraid.net/community/apps>`_. Uses Docker under the hood.
\ No newline at end of file
+You can install our :unraid:`unraid community apps <community/apps?q=ytdl-sub#r>` through the  `Unraid Community Apps plugin <https://unraid.net/community/apps>`_.
+
+
+After starting, if you installed the ``ytdl-sub-gui`` app, the code-server will be running at http://localhost:8443 (replace ``localhost`` with the IP of the computer running Unraid if you aren't trying to access ``ytdl-sub`` on that computer). Open this page in a browser to access and interact with ``ytdl-sub``. 
+
+At this time, we are unsure how to access the headless ``ytdl-sub`` console using the user ``abc`` and not ``root``. If you use Unraid and can help us figure this out, please contact the code owner or join the Discord.
```

Comment:
You should be able to do this via logging in as root (using the console button), then using the switch-user command `su abc`

Reply:
Okay it should be `su -l abc`

Reply:
Latest push updated to include headless access of Unraid app using `su abc -s /bin/bash`